src/meta-srv/src/handler/region_lease_handler.rs

```diff
@@ -73,15 +73,16 @@ impl HeartbeatHandler for RegionLeaseHandler {
             .region_lease_keeper
             .renew_region_leases(datanode_id, &regions)
             .await?;
+        let renewed_regions = renewed.keys().cloned();
+        let leader_regions = ctx.leader_region_registry.batch_get(renewed_regions);
 
         let renewed = renewed
             .into_iter()
             .map(|(region_id, region_role)| {
-                GrantedRegion {
-                    region_id,
-                    region_role,
-                }
-                .into()
+                let manifest_version = leader_regions
```

Comment:
We can make `stat.regions()` returns the `manifest_version`

Reply:
Actually, we can't rely on `stat.regions()` to provide the `manifest_version`, as it only contains the region manifest reported by the datanode. What we need here is the leader's manifest version.